lib/geordi/commands/dump.rb

```diff
@@ -5,18 +5,35 @@
 `geordi dump -l tmp/staging.dump` (with the `--load` option) sources the
 specified dump file into the development database.
 
+
+If you are using multiple databases per environment, pass the database name like this:
+
+    geordi dump -d primary
+
+Loading a dump into one of multiple local databases is not supported yet.
+
+When called with the `--load` option, sources the specified dump into the
+development database.
+
 `geordi dump staging` (with a Capistrano deploy target) remotely dumps the
 specified target's database and downloads it to `tmp/`.
 
+
 `geordi dump staging -l` (with a Capistrano deploy target and the `--load`
 option) sources the dump into the development database after downloading it.
 DESC
 
+
+option :load, aliases: ['-l'], type: :string, desc: 'Load a dump'
+option :select_server, default: false, type: :boolean, aliases: '-s'
+option :database, aliases: '-d', type: :string, desc: 'Specify key, under which database is nested in case of multiple', banner: 'DATABASE'
```

Comment:
Please change the description:

```suggestion
option :database, aliases: '-d', type: :string, desc: 'Database name, if there are multiple databases', banner: 'NAME'
```